enums/enums.js

```diff
@@ -146,6 +146,15 @@ function txTypeToGrpc(type) {
     }
 }
 
+function txStateToGrpc(state) {
+    switch(state) {
+        case TxState.MINED:     return 0
+        case TxState.PENDING:   return 1
+        case TxState.FAILED:    return 2
```

Comment:
Heavy smell, but you are the big boss

Reply:
![image](https://user-images.githubusercontent.com/13491551/93768644-ff8c7b00-fc19-11ea-8f89-84e72b6b1be5.png)